packages/devtools_app/lib/src/flutter/connect_screen.dart

```diff
@@ -104,25 +104,20 @@ class _ConnectScreenBodyState extends State<ConnectScreenBody> {
   }
 
   Future<void> _connect([_]) async {
-    var connected = false;
-    try {
-      connected = await FrameworkCore.initVmService(
-        '',
-        explicitUri: Uri.parse(controller.text),
-      );
-    } catch (_) {}
+    var uri = Uri.parse(controller.text);
+    final bool connected = await FrameworkCore.initVmService(
```

Comment:
this error reporting behavior is consistent with the dart:html app.

Reply:
Good to know.  Thanks for the API clarification.